sdk/inc/azure/iot/az_iot_adu_client.h

```diff
@@ -0,0 +1,620 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// SPDX-License-Identifier: MIT
+
+/**
+ * @file
+ *
+ * @brief Definition for the Azure IoT ADU Client
+ *
+ * @note More details about Azure Device Update can be found online
+ * at https://docs.microsoft.com/azure/iot-hub-device-update/understand-device-update
+ *
+ * @note You MUST NOT use any symbols (macros, functions, structures, enums, etc.)
+ * prefixed with an underscore ('_') directly in your application code. These symbols
+ * are part of Azure SDK's internal implementation; we do not document these symbols
+ * and they are subject to change in future versions of the SDK which would break your code.
+ *
+ */
+
+#ifndef _az_IOT_ADU_H
+#define _az_IOT_ADU_H
+
+#include <azure/core/az_json.h>
+#include <azure/core/az_result.h>
+#include <azure/core/az_span.h>
+#include <azure/iot/az_iot_hub_client.h>
+
+#include <stdbool.h>
+#include <stdint.h>
+
+#include <azure/core/_az_cfg_prefix.h>
+
+/**
+ * @brief  Define the ADU agent model ID.
+ */
+#define AZ_IOT_ADU_CLIENT_AGENT_MODEL_ID "dtmi:azure:iot:deviceUpdateModel;1"
+
+/**
+ * @brief ADU Agent Version
+ */
+#define AZ_IOT_ADU_CLIENT_AGENT_VERSION "DU;agent/0.8.0-rc1-public-preview"
```

Comment:
Should the agent version still be 0.8.0 RC1 public preview or a GA version?
/cc @wduraes

Reply:
this has been updated to GA